shared/container.go

```diff
@@ -406,6 +406,27 @@ func ConfigKeyChecker(key string) (func(value string) error, error) {
 	return nil, fmt.Errorf("Unknown configuration key: %s", key)
 }
 
+// KnownContainerDeviceKeys maps all fully defined, well-known device keys
+// to an appropriate checker function, which validates whether or not a
+// given value is syntactically legal.
+var KnownContainerDeviceKeys = map[string]func(value string) error{
+	"eth0.parent": IsAny,
```

Comment:
@stgraber can you tell me how do I find out all possible device keys?

Reply:
It's not really practical to do that I think because the actual list of valid keys depends on the type, subtype and a bunch of other options, so validating the query feels futile.

I'm actually surprised that we even bother to validate the `config` keys right now, I wasn't expecting that.

So for devices, I'd say, don't bother validating them, just try to get the value and if there's no such key, they'll get an empty string, that's fine.